client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.js

```diff
@@ -71,12 +71,52 @@ BreadcrumbLink.defaultProps = {
   isPublic: false
 };
 
+const RootProjectsLink = ({ api, section, operation, label }) => {
+  const dispatch = useDispatch();
+  const onClick = e => {
+    e.stopPropagation();
+    e.preventDefault();
+    dispatch({
+      type: 'DATA_FILES_SET_MODAL_PROPS',
+      payload: {
+        operation,
+        props: { showProjects: true }
+      }
+    });
+  };
```

Comment:
since this is only used if `section === 'modal'`, should we place it within that `if` statement?